src/components/ex-live-search-results/ex-live-search-results.scss

```diff
@@ -0,0 +1,101 @@
+.ex-live-search-results {
+  @apply overscroll-contain absolute hidden overflow-y-auto bg-white;
+
+  box-shadow: 0px 3px 45px #0000002c;
+  max-height: calc(100vh - 200px);
+  top: 100%;
+  width: 120%;
+
+  ul.ex-live-search-results__list {
+    @apply ml-0;
+  }
+
+  .ex-search-results-item {
+    @apply block clear-both px-5 pt-4 mt-0 overflow-hidden cursor-pointer;
+
+    &__media {
+      @apply float-right;
+
+      max-width: 150px;
+    }
+
+    &__topic {
+      @apply tracking-default text-xxs flex items-center mt-0 mb-1 font-light leading-tight text-black uppercase;
+    }
+
+    &__info {
+      @apply float-none;
+
+      overflow-wrap: break-word;
+      word-wrap: break-word;
+      -ms-word-break: break-all;
+      word-break: break-word;
+      -ms-hyphens: auto;
+      -moz-hyphens: auto;
+      -webkit-hyphens: auto;
+      hyphens: auto;
+    }
+
+    &__title {
+      @apply font-headline text-s mb-2 font-normal leading-4;
```

Comment:
Make the headline also bold like the regular search results.

![image](https://user-images.githubusercontent.com/88790097/141755801-ff6f13f1-d523-4cdf-a98e-3f81189b8cf8.png)

Reply:
made title bold